scripts/pr-ci-policy-bundle.cjs

```
@@ -0,0 +1,516 @@
+#!/usr/bin/env node
+'use strict';
+
+const crypto = require('node:crypto');
```

ESLint

[error] A `require()` style import is forbidden.

(`@typescript-eslint/no-require-imports`)

---

[error] 'require' is not defined.

(no-undef)